internal/indexmanagement/scripts.go

```diff
@@ -415,9 +415,11 @@ function getWriteAliases() {
   # "_cat/aliases/${policy}*-write?h=alias" | uniq
   aliasResponse="$(catWriteAliases "$policy")"
 
-  if [ -z "$aliasResponse" ]; then
+  if [ "$aliasResponse" = "error" ]; then
     echo "Received an empty response from elasticsearch -- server may not be ready"
     return 1
+  elif [ -z "$aliasResponse" ]; then
+    exit 0
```

Comment:
I think we can remove this, as the next operation is returning the empty response anyway.

```suggestion
```